classes/mock/generator.php

```diff
@@ -683,9 +687,14 @@ protected function getReflectionType(\reflectionMethod $method)
         return $this->hasReturnType($method) ? $method->getReturnType() : null;
     }
 
+    protected function getReflectionTypeName(\reflectionType $type)
+    {
+        return $type instanceof \reflectionNamedType ? $type->getName() : (string) $type;
+    }
+
     protected function isVoid(\reflectionMethod $method)
```

Comment:
Question: I got that this project uses a lowerCamelCase naming convention for *its own classes*, but why enforce it for *native PHP classes*? (for instance here `get_class($method)` would be `'ReflectionMethod'` i.e. UpperCamelCase) — and I see that `function isNullableParameter(\ReflectionParameter $parameter)` below uses the native casing 😕